gradle/enforcement/checkstyle.xml

```diff
@@ -222,24 +222,24 @@
     </module>
     <module name="ClassTypeParameterName">
       <!-- modified from default Google Java Style -->
-      <property name="format" value="^[A-Z]+[0-9]?$"/>
+      <property name="format" value="(^[A-Z][0-9]?)$|([A-Z][a-zA-Z0-9]*$)"/>
```

Comment:
I've never quite understood the standard styles related to type parameters - a type is a type. Can we loosen to that?

Reply:
I'm fine with both CamelCase and CONSTANT_CASE, as long as you can separate words in the type name - the current MORETHANONEWORDINATYPENAME makes two- or three-word type param names really hard to read.